Packs/FeedCyjax/Integrations/FeedCyjax/FeedCyjax.py

```diff
@@ -0,0 +1,629 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import urllib3
+import traceback
+import dateparser
+from datetime import datetime, timezone, timedelta
+from typing import Any, Dict, Tuple, List, Optional
+
+import cyjax as cyjax_sdk
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+INDICATORS_LAST_FETCH_KEY = 'last_fetch'
+INDICATORS_LIMIT = 50
+
+
+''' CLIENT CLASS '''
+
+
+class Client(object):
+    """Client class to interact with the Cyjax API using Cyjax SDK"""
+
+    def __init__(self, base_url, api_key, proxies=None, verify_ssl=True):
+        self.__base_url = base_url
+        self.__api_key = api_key
+        self.__proxies = proxies
+        self.__verify_ssl = verify_ssl
+        self._set_sdk()
+
+    def _set_sdk(self) -> None:
+        """Set Cyjax SDK
+        :return: None
+        """
+        cyjax_sdk.api_key = self.__api_key
+
+        if self.__base_url:
+            cyjax_sdk.api_url = self.__base_url
+
+        if self.__proxies:
+            cyjax_sdk.proxy_settings = self.__proxies
+
+        if self.__verify_ssl is False:
+            cyjax_sdk.verify_ssl = False
+
+    def test_connection(self) -> Tuple[bool, str]:
+        """Test connection to the Cyjax API using Cyjax SDK. Call indicator list API, and check if it's valid list
+
+        :return: A tuple with connection result and the error message if test failed.
+        :rtype: ``Tuple(bool, str)``
+        """
+        result = False
+        error_msg = 'Not responding'
+
+        try:
+            indicators = list(cyjax_sdk.IndicatorOfCompromise().list(since=timedelta(minutes=5)))
```

Comment:
Can you please explain the use for `5 minutes` here?

Reply:
It's a simple check If api/sdk returns correct list response. It might be empty or not I just check if it could connect to the API, and api returns correct response. Hence I don't check the indicators here but the api connection I just call the list endpoint with last 5 min filter, which in most scenario would return an empty list. 

I could replace since 5 min filter to limit=1 if you think that would be a better check.